### DISCUSSION TITLE
Can i use the language logo

### DISCUSSION THREAD

**Discussion by JASH-PATEL-6566** [rzashakeri/beautify-github-profile#6] [Q&A] [answered]
I am currently work on the website .... which is help to modify the github profile by click .... means u can write a code with only clicks .... so user can easily create profile ....so plz can i use the logo of language

**Comment by rzashakeri** [comment, marked as answer]
Hi My Friend 
yes, you can ✅

**Reply by JASH-PATEL-6566** [reply]
Thx man ...

**Reply by JASH-PATEL-6566** [reply]
But i am use in my personal project and i am host on google also .....
